src/lib/index.ts

```diff
@@ -3,6 +3,59 @@ import { MPlayerManager } from './mplayerManager'
 const DEFAULT_OP_TIMEOUT = 2000;
 const OPEN_OP_TIMEOUT = 8000;
 
+const parseKeyValueList = (data: string): { [id: string]: string } => {
```

Comment:
I would prefer to see these functions exported in a separate file just to keep things cleaner. This file is already quite large and could be split up a bit better.

Reply:
Additionally, please add some jsdoc comments on the new functions.

Reply:
:+1: